deepchem/models/torch_models/layers.py

```diff
@@ -1,6 +1,6 @@
 import math
 import numpy as np
-from typing import Any, Tuple, Optional, Sequence, List, Union
+from typing import Any, Tuple, Optional, Sequence, List, Union, Callable
```

Comment:
not needed

Reply:
Okay, I have used torch.nn.Module instead of Callable.